contracts/FeralfileArtworkV3.sol

```diff
@@ -136,10 +139,10 @@ contract FeralfileExhibitionV3 is ERC721Enumerable, Authorizable, IERC2981 {
         );
 
         Artwork memory artwork = Artwork(
-            title = title,
-            artistName = artistName,
-            fingerprint = fingerprint,
-            editionSize = editionSize
+            artwork_.title,
+            artwork_.artistName,
+            artwork_.fingerprint,
+            artwork_.editionSize
```

Comment:
Suggest to use keyword argument to prevent the variable position mismatching in the future.

Reply:
Fixed